includes/class-wc-payments-order-success-page.php

```diff
@@ -73,6 +75,58 @@ public function show_woopay_payment_method_name( $payment_method_title, $abstrac
 		return ob_get_clean();
 	}
 
+	/**
+	 * Add the BNPL logo to the payment method name on the order received page.
+	 *
+	 * @param string            $payment_method_title the default payment method title.
+	 * @param WC_Abstract_Order $abstract_order the order being shown.
+	 */
+	public function show_bnpl_payment_method_name( $payment_method_title, $abstract_order ) {
```

Comment:
This function shares a lot of code with the `show_woopay_payment_method_name()` function: did you consider refactoring to combine them or did it not appear practical? Just thinking out aloud.